README.md

```diff
@@ -73,16 +73,16 @@ fn main() {
 
 ## Let there be SparseSets
 
-I initially started to make an ECS to learn how it works. A failed attempt and some research later, I started to work on Shipyard.
+I initially started to make an ECS to learn how they work. After a failed attempt and some research, I started to work on Shipyard.
```

Comment:
I don't think "they" is correct. The subject here is "an ECS", singular.

An example sentence that might make it clearer:
"I received applications from many candidates, but then I interviewed Bob to see how ~they think~ he thinks"

I agree with the `After a` change, nice touch :)

Reply:
```suggestion
I initially started to make an ECS to learn how it works. After a failed attempt and some research, I started to work on Shipyard.
```